lib/destination/dml/merge.go

```diff
@@ -103,7 +103,7 @@ func (m *MergeArgument) GetParts() ([]string, error) {
 		equalitySQLParts = append(equalitySQLParts, equalitySQL)
 	}
 
-	cols := m.Columns.GetEscapedColumnsToUpdate(*m.UppercaseEscNames, m.DestKind)
+	cols := m.Columns.GetEscapedColumnsToUpdate(m.Dialect)
```

Comment:
We can remove `UppercaseEscNames` now I think

Reply:
Yep, I'm hoping so, was gonna do that in a follow up PR.

Reply:
Cool sgtm